packages/react/src/create-strict-api/index.ts

```diff
@@ -188,7 +204,10 @@ export interface CompiledAPI<TSchema extends CompiledSchemaShape> {
  * <div css={styles} />
  * ```
  */
-export function createStrictAPI<TSchema extends CompiledSchemaShape>(): CompiledAPI<TSchema> {
+export function createStrictAPI<
+  TSchema extends CompiledSchemaShape,
+  TCreateStrictAPIOptions extends StrictOptions = never
```

Comment:
This defaults to `never` else the union can't be effectively represented by TypeScript and it causes a type violation.